### PR TITLE
feat: use single vega binary & allow to change binary during node re-…

### LINKED ISSUE #266: Support protocol upgrade without visor
We have a requirement to test vega/data-node upgrades without visor. This means we need the ability to start/stop vega, and change the binary currently being used by different nodes in a vegacapsule network, at a time of our choosing (such that we could, for instance, stop 3/5 nodes, upgrade those, carry on for a time, and then later restart the other nodes with correct/incorrect versions to assert that the network behaves correctly.

### PR DESCRIPTION
**Pull request by karlem**
Close #266 and #259

### CODE REVIEW

File: cmd/install_dependencies.go [hunk 1]
```diff
@@ -11,7 +11,7 @@ import (
 )
 
 const (
-	latestReleaseTag = "v0.54.0"
+	latestReleaseTag = "v0.56.0"
```
Comment (jeremyletang) [inline cmd/install_dependencies.go:14]: 57?

File: installer/installer.go [hunk 2]
```diff
@@ -21,17 +21,13 @@ const (
 	repository      = "vega"
 	repositoryOwner = "vegaprotocol"
 
-	vegaBinName     = "vega"
-	walletBinName   = "vegawallet"
-	dataNodeBinName = "data-node"
+	vegaBinName = "vega"
```
Comment (daniel1302) [inline installer/installer.go:24]: I do not know if this has been introduced now or somewhere before but with default tag it is not working:

```

➜  vegacapsule git:(use-single-bin) vegacapsule install-bins --install-path ~/go/bin                              
Error: failed to install dependencies: failed to parse version from relase tag 
Usage:
  vegacapsule install-bins [flags]

Flags:
  -h, --help                         help for install-bins
      --install-path string          Install path for the binaries.
      --install-release-tag string   Automatically installs specific release tag version of vega, data-node and wallet binaries. (default "v0.57.0")

Global Flags:
      --home-path string   Specify the location of network home directory (default "/Users/daniel/.vegacapsule/testnet")
```


It is working when i specify flag explicitly:

```
➜  vegacapsule git:(use-single-bin) vegacapsule install-bins --install-path ~/go/bin --install-release-tag v0.57.0
2022/10/05 08:12:36 Starting to install binaries to "/Users/daniel/.vegacapsule/testnet/bins"
2022/10/05 08:12:36 Downloading release asset for "vega" with tag "v0.57.0"
```

**Review by daniel1302 — APPROVED** [review]

**Comment by daniel1302** [comment]
System-tests has been started against this PR here: https://jenkins.ops.vega.xyz/blue/organizations/jenkins/common%2Fsystem-tests/detail/system-tests/10572/pipeline/

**Comment by jgsbennett** [comment]
```
{
"system-tests":"capsule-single-bin-update"
}
```

**Pull request merged**